index.js

```diff
@@ -55,6 +55,8 @@ export default class WebTorrent extends EventEmitter {
     } else {
       this.peerId = arr2hex(text2arr(VERSION_PREFIX + arr2base(randomBytes(9))))
     }
+
+    /** @type {Uint8Array} */
     this.peerIdBuffer = hex2arr(this.peerId)
```

Comment:
This and `nodeIdBuffer` were being inferred as `Uint8Array` with a generic (not sure why), which is not valid and was causing an error.